### ISSUE TITLE
Accessibility: Exhibit Curation Box Headings

### ISSUE THREAD

**Issue by wrenaria** [UCDavisLibrary/ucdlib-wp-plugins#202]
Ideally you are not supposed to have "empty heading" on a webpage (headings without content under them).

Right now, the curation box on Exhibits has departments set as headings. While they are styled to look like heading text, they aren't actually functioning as such.

Functionally speaking, the "Exhibit from" and "Curated by" are working as headings, the department and curator names are content.  

Example from _[LGBTQ+ Health Book Exhibit](https://library.ucdavis.edu/exhibit/lgbtq-book-exhibit-bml/)_:
<img width="493" height="374" alt="Image" src="https://github.com/user-attachments/assets/7096ada0-9052-4443-a2a1-517526fa813c" />

When there are two departments listed, SiteImprove was able to notice and flag the problem due to two departments being treated as headings with nothing between them. **_SiteImprove [report 1](https://my2.siteimprove.com/Inspector/728315/A11Y/Page?pageId=247501747006&impmd=130BA75F00FDF9C4E13726DF81FAB862&issue=cantTell+failed&wcag=two#/sia-r78/failed/aXJNYlNOd3M=;OTM5MDA3NzE=), [report 2](https://my2.siteimprove.com/Inspector/728315/A11Y/Page?pageId=247476566977&impmd=130BA75F00FDF9C4E13726DF81FAB862&issue=failed+cantTell&wcag=two&decision=none#/sia-r78/failed)_**

Can you update the Exhibit content type to resolve this issue?

**Comment by spelkey-ucd** [comment]
Released [v4.5.0](https://github.com/UCDavisLibrary/ucdlib-wp-plugins/releases/tag/v4.5.0)
